cms/djangoapps/contentstore/views/course.py

```diff
@@ -1396,19 +1396,38 @@ def get_user_partition(self):
         return UserPartition.from_json(self.configuration)
 
     @staticmethod
-    def get_usage_info(course, store):
+    def _get_usage_info(course, unit, item, usage_info, group_id, scheme_name=None):
+        """
+        Get usage info for unit/module.
+        """
+        unit_url = reverse_usage_url(
+            'container_handler',
+            course.location.course_key.make_usage_key(unit.location.block_type, unit.location.name)
+        )
+
+        usage_dict = {'label': u"{} / {}".format(unit.display_name, item.display_name), 'url': unit_url}
+        if scheme_name == UserPartition.RANDOM_SCHEME:
```

Comment:
Why do we only want to show validation messages for split_test instances? I would vote for removing this conditional.